roles/template_service_broker/vars/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
-__tsb_files_location: "../../../files/origin-components/"
+__tsb_files_location: "../files/"
```

Comment:
If the files are moving into `<role>/files`, this var is no longer necessary to explicitly define the path.  The two `copy` tasks using this var should be updated.

http://docs.ansible.com/ansible/latest/playbooks_reuse_roles.html#using-roles
> Any copy, script, template or include tasks (in the role) can reference files in roles/x/{files,templates,tasks}/ (dir depends on task) without having to path them relatively or absolutely.

Reply:
+1